### PR TITLE
Commerce 1648 Minium Login CSS fixes and add redirect to private pages

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

File: commerce-theme-minium/commerce-theme-minium/src/css/_login.scss [hunk 1]
```diff
@@ -116,6 +124,30 @@
                     margin-bottom: 0;
                 }
             }
+
+            .col-md-12.portlet-column.portlet-column-only {
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/_login.scss:109]: selectors should be sorted alphabetically -- so this should be moved up above `.navigation {`

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium/src/css/_login.scss [hunk 2]
```diff
@@ -60,7 +60,7 @@
         }
     }
 
-    #_com_liferay_login_web_portlet_LoginPortlet_loginForm {
+    #_com_liferay_login_web_portlet_LoginPortlet_loginForm, #_com_liferay_login_web_portlet_LoginPortlet_fm {
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/_login.scss]: technically this comma-delimited list of selectors should be alpha sorted as well :(. Sorry, missed it the first time around.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
@iovaneale new commit just for you ;) :heart:

File: commerce-theme-minium/commerce-theme-minium/src/css/_login.scss (hunk 2 above)
Comment (ethanbustad) [reply commerce-theme-minium/commerce-theme-minium/src/css/_login.scss]: @alecsloan this should still be fixed

Comment (alecsloan) [reply commerce-theme-minium/commerce-theme-minium/src/css/_login.scss]: Updated, thanks!

**Pull request merged**